### PR TITLE
Linking independent pages to form complete UI

### PR DESCRIPTION
**Pull request by LawreenL**
added links to all paths across log in -> dashboard -> issue/return lists -> issue/return pages.
moved reserves.html and searches.html to manager and client respectively. 
Currently don't have access to the home page so could not link that yet.

### CODE REVIEW

File: manager/reserve.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
               <div class="row">
                 <div class="col s1">
-                    <a href="issue.html" class="brand-logo">DAM</a>
+                    <a href="/prototype/dashboard.html" class="brand-logo">DAM</a>
```
Comment (anandimous) [inline manager/reserve.html]: change this to ../prototype/dashboard.html

File: manager/return_list.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        <title>DAM | Authorize Reserves</title>
+    </head>
+
+    <body>
+        <nav>
+          <div class="nav-wrapper">
+
+              <div class="row">
+                <div class="col s1">
+                    <a href="/prototype/dashboard.html" class="brand-logo">DAM</a>
```
Comment (anandimous) [inline manager/return_list.html]: change this to ../prototype/dashboard.html

Comment (LawreenL) [reply manager/return_list.html]: done!

File: manager/reserve.html (hunk 1 above)
Comment (LawreenL) [reply manager/reserve.html]: done!

File: prototype/dashboard.html [hunk 3]
```diff
@@ -34,7 +34,20 @@
   <header>
     <nav>
       <div class="nav-wrapper">
-        <a href="#" class="brand-logo">DAM</a>
+          <div class="row">
+            <div class="col s1">
+                <a href="/prototype/dashboard.html" class="brand-logo">DAM</a>
```
Comment (anandimous) [inline prototype/dashboard.html]: Here too, needs to be ../prototype/dashboard.html

Comment (LawreenL) [reply prototype/dashboard.html]: yikes! Double-checked every page again as well just to be sure!

**Review by lawzeem — APPROVED** [review]
LGTM

**Pull request merged**